Dockerfile

```diff
@@ -36,17 +36,11 @@ ENV JAVA_HOME=/opt/jdk-mini
 ENV PATH="$PATH:$JAVA_HOME/bin"
 ENV DIRPATH /pitchfork
```

Comment:
this can be ARG so it's not used as env var

Reply:
yeap, i will change that